openapi.yaml

```diff
@@ -330,6 +330,10 @@ definitions:
         type: boolean
         description: Advance standing indicator
         example: true
+      mandChecklistInd:
+        type: string
+        description: Mandatory Checklist Indicator
+        example: 'N'
```

Comment:
Is `MAND_CHECKLIST_IND` similar to `ASW_IND` where it should just either be `'Y'` or `'N'`? If so, would it make more sense to make `mandChecklistInd` a boolean like `aswInd`? If this property can have other string values or if it needs to be a string in the request for some other reason, I think we could add a `pattern` here to limit the allowed values for it. If we don't know all the allowed values, we could add `minLength`/`maxLength` here since it appears to be a `VARCHAR2(1)`.

Reply:
Good catch. I confirmed with Lance it is either 'Y', 'N', or null so I made it a boolean